src/utils/components/UserCredentials/useSSHCommand.ts

```diff
@@ -0,0 +1,46 @@
+import { InfrastructureModel, modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Service } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getCloudInitCredentials } from '@kubevirt-utils/resources/vmi';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+export type useSSHCommandResult = {
+  command: string;
+  user: string;
+  sshServiceRunning: boolean;
+};
+
+const getInfrastructureAPIURL = (infrastructure: any): string =>
```

Comment:
can be in a util file , also do we really don't have any other options than : any ?

Reply:
I've found only `K8sResourceKind` and `any` in the console code. But with `K8sResourceKind` i can't do `infrastructure.status`

Reply:
if we have InfrastructureModel we don't have a matching API? for example V1Infrastructure or something like that?